app/models/account/provider_methods.rb

```diff
@@ -334,7 +334,8 @@ def create_first_service
       plan.create_contract_with!(self)
     end
 
-    services.create! name: "API"
+    creator = ServiceCreator.new(service: services.build(name: 'API'))
+    creator.call!
```

Comment:
If it is considered simple enough in one line in [ProviderSignup](https://github.com/3scale/porta/pull/1211/files#diff-ba591bb65af24eb68a38faa20a776e17R67), then probably also here :thinking: